CenterForOpenScience/osf.io
# [feature] Fix fork count [OSF-8422]

## Purpose
Fork counts include registrations of forks. This makes the count on the forks page different than the number shown.


## Changes
Make the query used to display forks more similar to the fork_count query.


## Side effects
I added a space between View forks and the count, because why no space...
Before:
<img width="183" alt="screen shot 2017-08-11 at 4 37 13 pm" src="https://user-images.githubusercontent.com/1322421/29231092-d9a15f0a-7eb3-11e7-901e-4333e2e96644.png">
After:
<img width="191" alt="screen shot 2017-08-11 at 4 41 15 pm" src="https://user-images.githubusercontent.com/1322421/29231112-efde2b86-7eb3-11e7-8962-5d01c28e7433.png">
#scopecreep



## Ticket

https://openscience.atlassian.net/browse/OSF-8422

## QA Notes
Make fork, register fork, make sure registered fork no longer increases the count and that the number of forks on the view page matches the number displayed on the dropdown.
